### DISCUSSION TITLE
Functionality

### DISCUSSION THREAD

**Discussion by WyattGilette87** [tagyoureit/nodejs-poolController#865] [General]
I've gotten the pool control and dashboard set up and I'm very impressed! This is so sweet. I am looking at this system to use to control an aqua culture system. The only things it lacks that i need are as follows: 1) most importantly intercession of a salinity sensor that can read in parts per thousands( atlas scientific,) makes this sensor. 2) a dissolved oxygen sensor which atlas also sells..... Can anyone here help me make my dreams come true?

I'm making a closed system to breed oysters

**Comment by katodude** [comment]
Check out the REM. It has support the the Atlas sensors.

**Comment by rstrouse** [comment]
> I'm making a closed system to breed oysters

You would think getting oysters in the mood would be really easy.

REM has support for the EZO Conductivity circuit.  We didn't do anything with the EZO DO.

**Comment by WyattGilette87** [comment]
What would it take to get that programming added?

**Reply by rstrouse** [reply]
Probably not much as it really only reads and does not have any Calibration commands.

**Comment by WyattGilette87** [comment]
Who do I have to bribe LOL

**Comment by WyattGilette87** [comment]
Is anything here adaptable to nodejs- pool controller

https://github.com/jvsalo/atlas_scientific

**Reply by rstrouse** [reply]
Nope that is simply calling the cli.  REM polls the data out of I2c and feeds it over a socket, webhook, or MQTT.

**Comment by WyattGilette87** [comment]
Ok
